WordPress/src/main/java/org/wordpress/android/ui/uploads/UploadService.java

```diff
@@ -403,6 +403,10 @@ public static boolean hasMediaErrorForPost(PostModel post) {
         return error != null && error.mediaError != null;
     }
 
+    public static List<MediaModel> getAllInProgressOrQueuedMediaItemsForPost(PostModel post){
```

Comment:
This method could be renamed to better match `MediaUploadHandler.getPendingOrInProgressMediaUploadsForPost()`. It's a bit confusing to refer to the media as "in progress or queued" in one place and "pending or in progress" in another.